mne/decoding/transformer.py

```diff
@@ -17,7 +18,10 @@
 
 
 class Scaler(TransformerMixin):
-    """Standardize data across channels.
+    u"""Standardize data across channels.
+
+    By default, this makes each time point (within each epoch) have
```

Comment:
@jasmainak This is an independent issue IMO. 

#3672 is supposed to take care of this by using robust scaler instead of the z-score.

I suggest we move this discussion & correction to #3672; this PR is about updating the BaseEstimator.